.github/workflows/deploy-release.yml

```
@@ -21,14 +21,29 @@ jobs:
     - name: Build a wheel and a source tarball
       run: |
         python -m build --sdist --wheel --outdir dist/
-    - name: Publish distribution to Test PyPI
-      if: ${{ github.event_name == 'release' }}
-      uses: pypa/gh-action-pypi-publish@release/v1
-      with:
-        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
-        repository_url: https://test.pypi.org/legacy/
+  publish-pypi:
+    name: Publish Python distribution to test.pypi.org
```

```suggestion
    name: Publish Python distribution to pypi.org
```

It seems to me that this one is for the main PyPI.